x/blob/types/payforblob.go

```diff
@@ -185,8 +184,8 @@ func CreateCommitment(blob *Blob) ([]byte, error) {
 	// the commitment is the root of a merkle mountain range with max tree size
 	// equal to the minimum square size the blob can be included in. See
 	// https://github.com/celestiaorg/celestia-app/blob/fbfbf111bcaa056e53b0bc54d327587dee11a945/docs/architecture/adr-008-blocksize-independent-commitment.md
```

Comment:
[blocking] this comment needs to be updated

Reply:
https://github.com/celestiaorg/celestia-app/pull/1604/commits/6b0353f84554a6c4a711d37b6d2e33ca40db957c